content/zh-cn/blog/_posts/2025-08-27-volume-group-snapshot-beta-2.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: blog
+title: "Kubernetes v1.34: 将卷组快照推进至 v1beta2 阶段"
+draft: true
+date: 2025-XX-XX
```

Comment:
This date and filename has been changed in https://github.com/kubernetes/website/pull/52157